.github/workflows/pytest.yml

```diff
@@ -0,0 +1,43 @@
+name: Pytest
+
+on:
+  pull_request:
+    branches:
+      - master
+      - main
+      - dev
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+
+    steps:
+    - name: Checkout code
+      uses: actions/checkout@v3
+
+    - name: Set up Python
+      uses: actions/setup-python@v4
+      with:
+        python-version: "3.10"
+
+    - name: Install dependencies
+      run: |
+        python -m venv .venv
+        source .venv/bin/activate
+        pip install -r requirements.txt
+        pip install pytest
+
+    - name: Run all tests except cache
+      run: |
+        source .venv/bin/activate
+        pytest tests/ --ignore tests/test_cache.py
```

Comment:
Also exclude `test_price_repair.py`. It does lots of fetches, and shouldn't be affected by most PRs. I run repair tests manually.